assets/build/store.js

```
@@ -1 +1,1228 @@
+        ...state,
+        needSorting: action.needSorting
+      };
+    case 'SET_HELPFUL_DOCS':
+      return {
+        ...state,
+        helpfulDocs: action.helpfulDocs
+      };
+    case 'REMOVE_DOC':
+      return {
+        ...state,
+        docs: [...state.docs?.filter(doc => doc.id !== action.docId)],
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)